docs/Release_Notes.txt

```diff
@@ -0,0 +1,497 @@
+  parm/prepobs_cqcbufr.gfs.parm
+  parm/prepobs_cqcbufr.nam.parm
+  parm/prepobs_cqcbufr.rap.parm
+  parm/prepobs_prepacqc.cdas.parm
+  parm/prepobs_prepacqc.gdas.parm
+  parm/prepobs_prepacqc.gfs.parm
+  parm/prepobs_prepacqc.nam.parm
+  parm/prepobs_prepacqc.rap.parm
+  parm/prepobs_prepdata.cdas.parm
+  parm/prepobs_prepdata.gdas.parm
+  parm/prepobs_prepdata.gfs.parm
+  parm/prepobs_prepdata.nam.parm
+  parm/prepobs_prepdata.rap.parm
+  parm/prepobs_prepdata.rtma.parm
+  parm/prepobs_prepdata.urma.parm
+  parm/prepobs_prepssmi.cdas.parm
+  parm/prepobs_prepssmi.gdas.parm
+  parm/prepobs_prepssmi.gfs.parm
+  parm/prepobs_prepssmi.nam.parm
+  parm/prepobs_prepssmi.rap.parm
+  parm/prepobs_prepssmi.rtma.parm
+  parm/prepobs_prevents.cdas.parm
+  parm/prepobs_profcqc.gdas.parm
+  parm/prepobs_profcqc.gfs.parm
+  parm/prepobs_profcqc.nam.parm
+  parm/prepobs_profcqc.rap.parm
+  parm/syndat_syndata.gdas.parm
+  parm/syndat_syndata.gfs.parm
+  parm/syndat_syndata.nam.parm
+  parm/transfer_airnow.list
+  parm/transfer_hourly.list.old0203
+  parm/wave_dcodquikscat.rtma.parm
+A release_notes/CMakeLists.txt
+A release_notes/Release_Notes.txt
+  scripts/CMakeLists.txt
+  scripts/exairnow_dump.sh
+M scripts/excdas_dump.sh
+  scripts/excdas_makeprepbufr.sh
+  scripts/exdump_alert.sh
+  scripts/exdump_monitor.sh
+  scripts/exdump_post.sh
+M scripts/exglobal_dump.sh
+M scripts/exglobal_makeprepbufr.sh
+  scripts/exmods.sh
+  scripts/exnam_dump.sh
+M scripts/exnam_makeprepbufr.sh
+  scripts/exprep_post.sh
+M scripts/exrap_dump.sh
+  scripts/exrap_makeprepbufr.sh
+M scripts/exrtma_dump.sh
+  scripts/exrtma_makeprepbufr.sh
+M scripts/exurma_dump.sh
+  scripts/exurma_makeprepbufr.sh
+  sorc/CMakeLists.txt
+  sorc/bufr_avgdata.fd/CMakeLists.txt
+  sorc/bufr_avgdata.fd/avgdata.f
+  sorc/bufr_datacount.fd/CMakeLists.txt
+  sorc/bufr_datacount.fd/datacount.f
+  sorc/bufr_listdumps.fd/CMakeLists.txt
+  sorc/bufr_listdumps.fd/inc_w3unpkb7.inc
+  sorc/bufr_listdumps.fd/iw3unpbf.f
+  sorc/bufr_listdumps.fd/listdumps.f
+  sorc/bufr_listdumps.fd/w3unpkb7.f
+  sorc/bufr_listdumps.fd/wordlength.f
+  sorc/bufr_prepmods.fd/CMakeLists.txt
+  sorc/bufr_prepmods.fd/bufr_prepmods.f
+  sorc/bufr_prepmods.fd/clonlat.f
+  sorc/bufr_prepmods.fd/mbuoyb.f
+  sorc/bufr_prepmods.fd/mnemonics.f
+  sorc/bufr_prepmods.fd/obname.f
+  sorc/bufr_prepmods.fd/raddate.f
+  sorc/bufr_prepmods.fd/ssmipn.f
+  sorc/bufr_prepmods.fd/tideg.f
+  sorc/bufr_remorest.fd/CMakeLists.txt
+  sorc/bufr_remorest.fd/remorest.f
+  sorc/prepobs_prepanow.fd/CMakeLists.txt
+  sorc/prepobs_prepanow.fd/prepanow.f
+  triggers/ (30)
+  ush/CMakeLists.txt
+A ush/build.sh
+  ush/bufr_avgdata.sh
+  ush/bufr_datacount.sh
+  ush/bufr_prepmods.sh
+  ush/bufr_remorest.sh
+  ush/dumpmb
+  versions/README
+  versions/build.ver
+M versions/run.ver
+M versions/VERSION
+------------------
+M-modified; A-added; D-deleted
+
+JOBS file changes
+-----------------
+ jobs/JOBSPROC_AIRNOW_DUMP
+ Correct COMIN path 
```

Comment:
This (and similar information below) belongs in a ChangeLog, not release note.

Reply:
respectfully, I find this comment contradicting your previous comment on what RN should contain